Runtime/Visual/PointsRenderer.cs

```diff
@@ -20,38 +22,53 @@ public class PointsData
             /// <summary>
             /// Represents the start, i.e. the first rendered point.
             /// </summary>
-            public GameObject start;
+            [Serialized]
+            public GameObject PointStart { get; set; }

```

Comment:
Changing all of these new names to have `Point` at the end reads better to me. `StartPoint`, `RepeatedSegmentPoint`, `EndPoint`.